Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -1319,6 +1327,27 @@ namespace AtomSampleViewer
         s_instance->m_scriptOperations.push(AZStd::move(operation));
     }
 
+    void ScriptManager::Script_CaptureCpuFrameTime(AZ::ScriptDataContext& dc)
```

Comment:
should this be const AZ::ScriptDataContext& dc?

Reply:
I know other script functions do not declare this as const, just wondering if it ought to be.